package.json

```diff
@@ -18,14 +20,17 @@
   },
   "homepage": "https://github.com/natemoo-re/astro-remote#readme",
   "volta": {
-    "node": "18.16.0"
+    "node": "18.19.0",
+    "pnpm": "8.14.1"
   },
-  "packageManager": "pnpm@7.6.0",
-  "dependencies": {
-    "@changesets/cli": "^2.18.1",
-    "@changesets/changelog-github": "^0.4.2"
+  "engines": {
+    "node": ">=18.19.0"
```

Comment:
This would be something to callout in the changelog. Adding `engines` is always a breaking change, and since we're doing a breaking change we might want to require Node 20 since it's the current LTS release?